orchestration/dagster_orchestration/hca_orchestration/pipelines/load_hca.py

```diff
@@ -72,17 +83,96 @@
         "load_tag": load_tag,
         "scratch_config": scratch_config,
         "target_hca_dataset": target_hca_dataset,
-        "bigquery_service": mock_bigquery_service
+        "bigquery_service": mock_bigquery_service,
+        "slack": console_slack_client,
+        "dagit_config": preconfigure_resource_for_mode(dagit_config, "test")
+    }
+)
+
+
+def _base_slack_blocks(title: str, key_values: dict[str, str]) -> list[dict[str, object]]:
```

Comment:
🔧 This is a copypasta of this [code](https://github.com/DataBiosphere/hca-ingest/blob/09fcd224c8638a2a059f2ac8244c7ff7f438893e/orchestration/dagster_orchestration/hca_orchestration/pipelines/cut_snapshot.py#L90), what do you think about extracting a common method?

Reply:
If I abstract this within the HCA project, should it stay within the hca_orchestration project/directory? My initial thought was to add it to the [dagster_utils slack resource](https://github.com/broadinstitute/dagster-utils/blob/main/dagster_utils/resources/slack.py) but I'm wondering if this would be usable across all projects pulling from dagster_utils or if this is specifically for HCA?

Reply:
🤔 I think let's leave it in HCA for now.